validation/standard/standard-bytecodes.toml

```diff
@@ -1,3 +1,23 @@
+["op-contracts/v1.8.0"] # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
+# Updated in this release
+system_config = "0x165dece6291f3a071e602c804f4a3e89977a71ec866f46c85eb21448f22f6f9a"             # version 2.3.0
+fault_dispute_game = "0x1a81c5fe9fbbf68a0f75e987451e5181c4fc3e801f5c056d38f08942cefed349"        # version "1.3.1"
+permissioned_dispute_game = "0x69bd3197f9f0248972d9518a9c220a22e326760dc8c0ffcc1612d4d0730a7a1a" # version "1.3.1"
+mips = "0xd80e53cfb3a0655e5a82b40f370b7352bd926d6d28de7f1735583fcfeba3f0fd"                      # version "1.2.1"
+# Unchanged in this release
+address_manager = ""
+l1_cross_domain_messenger = "0xc919b535f7bbbaa57f32b602f26000e3997c22c933efbb92a36e053b20b2b4b1"       # version 2.3.0
+l1_erc721_bridge = "0xa1a3ea3996797cf5035dafff9335386f8b00c58ff22aaa4c34b02c4ae7cf5e8a"                # version 2.1.0
+l1_standard_bridge = "0x6894839591ba2aa1f6ba43526113e25fd795ec5c5a48202e664c931cd170aaee"              # version 2.1.0
+optimism_mintable_erc20_factory = "0xb8ec72533005496795f8fc0c71f0312578ff626f7b94699984e1add6dc2f3b88" # version 1.9.0
+optimism_portal = "0x2b0610230634fccf9d05f4aaefceffef9e04aa99910798d42823e0068eb07eca"                 # version 3.10.0
+preimage_oracle = "0x2109eb20fb34703fe7ab0231bf0e93d972fe78423b72e7fb1f797f168217b87d"                 # version "1.0.0"
+# for the contracts below which contain immutables, the hash is calculated from the masked bytecode (not the deployed bytecode as-is).
+# sections of the bytecode that contain the immutables are masked (set to 0) before computing the hash
+anchor_state_registry = "0x46e8ecc46fd27f178ada1c5a2168c09512e7ed2f7d4ac9a7c74123af26fd850c" # version "2.0.0"
+delayed_weth = "0xc4fb1a6fa54865937fda773735938b5374d23fcf556bb4c69686f23be96b3f97"          # version "1.0.0"
```

Comment:
further down you reference `v1.1.0`

Reply:
[15e65ac](https://github.com/ethereum-optimism/superchain-registry/pull/704/commits/15e65acbebcd0936e92807b61c1b95c0e87ddc83)